src/docker-compose.override.yml

```diff
@@ -15,6 +15,7 @@ services:
       - PROMITOR_INTERNAL_CONFIG_FOLDER=/config/
       - PROMITOR_AUTH_APPID=ceb249a3-44ce-4c90-8863-6776336f5b7e
       - PROMITOR_AUTH_APPKEY=ZgLy6zYNh9SEmIl0B+rv+ZuQQ2wJyQi/tTXnp2Wp9PM=
+      - PROMITOR_ATLASSIAN_STATUSPAGE_APIKEY=53dbaf1c-8551-48ef-81f9-481eb3ac64cb
```

Comment:
Just wanna double check - is this just a dummy value or is it your real API key? If it's real, should it be committed or should this file be gitignored?

Reply:
Dummy value for sure 🤡

Reply:
Thanks for noting